Algebraic Topology/algebraic-topology.tex

```diff
@@ -454,9 +454,94 @@ \section{Functoriality of the fundamental group}
 \end{proof}
 
 
+\mydate{d7}{8}{2}{2017}
 
+\section{CW complexes and the fundamental group}
 
+Consider the polygon
 
+\begin{figure}
+  % \includegraphics{}
+  \label{fig:star}
+\end{figure}
+
+$X_0$ = picture
+
+$X_1$ = picture ((it seems I am inconsistent with whether these are capital x's or lowercase.  Should they all be capital?  I went back and made them all capital.))
+
+\begin{thm}
+  If $X$ is a CW complex and $A$ a contractible subcomplex, then $X$ is \textbf{not} equal to $X/A$ (this is the quotient by the equivalence relation $x \sim x'$ iff $x = x'$ or $x, x' \in A$).
```

Comment:
I think the point of this theorem is that $X$ is homotopy equivalent to X/A (as you define it).